Examples/base-py/Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM mcr.microsoft.com/aiforearth/base-py:1.5
-
-# Example of installing additonal Anaconda packages - numpy and pandas
-RUN echo "source activate ai4e_py_api" >> ~/.bashrc \
-    && conda install -c conda-forge -n ai4e_py_api numpy pandas
```

Comment:
Actually one more question - it'd be good to show how to install additional pip and conda packages. Is conda still available?

Reply:
The API bases have been converted to use pip for all installs, so Conda is not needed.  I added a pip install example in this API.